### ISSUE TITLE
Issue with copying hashtags that are in a post

### ISSUE THREAD

**Issue by Zero3K** [bluesky-social/social-app#6863]
### Steps to Reproduce

1. Copy the text and hashtag(s) of a post that contain both.
2. Paste it into a program such as Notepad.

### Attachments

https://github.com/user-attachments/assets/0a2f5e8d-61b4-4255-8c5f-1702c1b01a31

### What platform(s) does this occur on?

Web (Desktop)

### Device Info

Operating System: Windows 11

### What version of the app are you using?

1.95.0

**Comment by surfdude29** [comment]
Are you able to reproduce this when using the **Copy post text** option from the post context menu? That might be a better option.

**Comment by Zero3K** [comment]
No, but that involves having to click something that I am not used to clicking.

**Comment by c960657** [comment]
I don't know if it has any bad side-effects, but I think this issue can be fixed by replacing `<View>` with `<Text>` in [EventStopper.tsx](https://github.com/bluesky-social/social-app/blob/main/src/view/com/util/EventStopper.tsx).

**Comment by Zero3K** [comment]
Can someone please make a PR to see if doing that fixes this issue?

**Comment by c960657** [comment]
It seems you also need to add `display: 'inline'` to `triggerStyle` in the `<NativeDropdown>` tag in `TagMenu/index.web.tsx` to make it work in Chrome.

**Comment by Zero3K** [comment]
Any news regarding this?

**Comment by mozzius** [comment]
Looks like this was fixed? Closing

**Comment by Zero3K** [comment]
I can confirm that it has been fixed a while ago.
